packages/core/src/api/events/types.ts

```diff
@@ -0,0 +1,16 @@
+import type { TraceContext } from '../traces';
+
+export type EventAttributes = Record<string, string>;
+
+export interface EventEvent {
```

Comment:
nit: Could we maybe use AgentEvent?

Reply:
Agreed it's a bit confusing but EventEvent might be the best option considering the other types follow the pattern <type>Event like TraceEvent or LogEvent

Reply:
That's what I thought - there's no great options here, so let's stick to the convention